test/extended/imageapis/quota_admission.go

```diff
@@ -4,33 +4,32 @@ import (
 	"fmt"
 	"time"
 
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kutilerrors "k8s.io/apimachinery/pkg/util/errors"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
 
 	g "github.com/onsi/ginkgo"
 	o "github.com/onsi/gomega"
 
+	imageapiv1 "github.com/openshift/api/image/v1"
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
-	imagesutil "github.com/openshift/origin/test/extended/images"
 	exutil "github.com/openshift/origin/test/extended/util"
 	testutil "github.com/openshift/origin/test/util"
 )
 
 const (
-	imageSize = 100
-
-	quotaName = "isquota"
-
+	quotaName   = "isquota"
 	waitTimeout = time.Second * 600
 )
 
-var _ = g.Describe("[Feature:ImageQuota][registry][Serial][Suite:openshift/registry/serial][local] Image resource quota", func() {
+var _ = g.Describe("[Feature:ImageQuota] Image resource quota", func() {
```

Comment:
I think this means the test will get run in conformance, but did you confirm you saw it run in one of the job outputs?

Reply:
I'm not seeing it listed in either e2e-gcp or e2e-gcp-image-registry.

Reply:
I see...  well assuming this test works (because image quota is on by default now?) let's re-enable it.  If it still can't be run, i guess this is fine.

I see we have another test in the same boat (limitrange_admission).